packages/turf-interpolate/index.js

```diff
@@ -49,7 +51,7 @@ export default function (points, cellSize, gridType, property, units, weight) {
     switch (gridType) {
     case 'point':
     case 'points':
-        grid = poinGrid(box, cellSize, units, true);
+        grid = poinGrid(box, cellSize, {units: units, bboxIsMask: true});
```

Comment:
@stebogit We should enforce strict error catching to prevent forgetting adding extra optional params.

Reply:
aaaaah, that's why! great job! 👍

Reply:
Sooooo annoying because there was no errors causing this issue